qtconsole/rich_jupyter_widget.py

```diff
@@ -1,7 +1,7 @@
 # Copyright (c) IPython Development Team.
 # Distributed under the terms of the Modified BSD License.
 
-from base64 import decodestring
+from base64 import decodebytes
```

Comment:
``decodestring`` has been deprecated since Python 3.1 (released nearly 10 years ago). ``decodebytes`` is the direct replacement, although its still considered part of the "legacy" interface of the module. We could also consider just swapping it for the modern (but still present on Python 2) ``base64.b64decode()`` interface, which should be functionally identical here.

Reply:
I ended up doing so, since looking into it seems ``decodebytes`` doesn't exist on Python 2, while their "modern" equivalents do. Running the Spyder test suite against these two changes now...